Makefile

```diff
@@ -14,10 +16,8 @@ login-vancouver:
 login-sandbox:
 	fly -t sandbox login  --concourse-url http://192.168.100.4:8080/
 
-pipeline-master:
-	yes | fly -t ${TARGET} set-pipeline -c bosh-linux-stemcell-builder-master.yml -p stemcells-master -l ../cloudfoundry/secure/concourse-secrets.yml
-	fly -t ${TARGET} unpause-pipeline -p stemcells-master
-
-pipeline-check:
-	yes | fly -t ${TARGET} set-pipeline -c bosh-linux-stemcell-builder-check.yml -p stemcells-check -l ../cloudfoundry/secure/concourse-secrets.yml
-	fly -t ${TARGET} unpause-pipeline -p stemcells-check
+pipeline-%:
+	gpg --decrypt --batch --no-tty ${CONCOURSE_SECRETS_FILE} > ${TMP_SECRETS_FILE} 2> /dev/null
+	yes | fly -t ${TARGET} set-pipeline -c bosh-linux-stemcell-builder-$*.yml -p stemcells-$* -l ${TMP_SECRETS_FILE}
+	fly -t ${TARGET} unpause-pipeline -p stemcells-$*
+	rm -f ${TMP_SECRETS_FILE}
```

Comment:
This means that if we had a problem with `fly` (e.g. intermittent connection issues) we'll leave a decrypted secrets file on disk (because make aborts after first error). That doesn't sound very awesome.